code/datums/wires/wires.dm

```diff
@@ -308,22 +310,30 @@ var/list/wire_color_directory = list()
 /datum/wires/ui_act(action, params)
 	if(..() || !interactable(usr))
 		return
+	if(wireused)
+		return
 	var/target_wire = params["wire"]
 	var/mob/living/L = usr
 	var/obj/item/I = L.get_active_hand()
 	switch(action)
 		if("cut")
 			if(istype(I, /obj/item/weapon/wirecutters) || IsAdminGhost(usr))
 				playsound(holder, 'sound/items/Wirecutter.ogg', 20, 1)
+				wireused = 1
 				cut_color(target_wire)
 				. = TRUE
+				spawn(wire_cooldown)
+					wireused = 0
 			else
 				L << "<span class='warning'>You need wirecutters!</span>"
 		if("pulse")
 			if(istype(I, /obj/item/device/multitool) || IsAdminGhost(usr))
 				playsound(holder, 'sound/weapons/empty.ogg', 20, 1)
+				wireused = 1
 				pulse_color(target_wire)
 				. = TRUE
+				spawn(wire_cooldown)
```

Comment:
Actually, you can just move this code outside the if checks to avoid having to copypaste it in each option. You should do that.